### PR TITLE
Remove local server checkbox from preferences

### PR DESCRIPTION
**Pull request by CapnCheapo**
First contribution :)    

This is probably a work in progress and needs further testing.

Removed Use Local server checkbox as discussed in #1682.  
Also added my name to the developers list and fixed a couple of grammar errors.

### CODE REVIEW

File: gns3/modules/qemu/pages/qemu_preferences_page.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def _populateWidgets(self, settings):
         :param settings: QEMU settings
         """
 
-        self.uiUseLocalServercheckBox.setChecked(settings["use_local_server"])
+        #self.uiUseLocalServercheckBox.setChecked(settings["use_local_server"])
```
Comment (noplay) [inline gns3/modules/qemu/pages/qemu_preferences_page.py]: We could just drop the line to keep code clean

**Comment by noplay** [comment]
Also do you agree to sign the Contributor Licence Agreement?
https://github.com/GNS3/cla

We at GNS3 are eager to work with you. For small changes — little bugfixes, correcting typos, and the like — please just submit pull requests to any of our projects. For larger changes, though, we have to ask you to jump through a little hoop.

In particular, in order for us to accept any major patches from you, you will have to electronically sign a statement that indicates two things:

You are willingly licensing your contributions under the terms of the open source license of the project that you’re contributing to.
You are legally able to license your contributions as stated.
The reason we do this is to ensure, to the extent possible, that we don’t “taint” the projects we manage with contributions that turn out to be improper. This protects everyone who wants to use the projects, including you!

Once you sign the Contributor License Agreement (the “CLA”), we will then be able to merge your contributions with a clear conscience and with only the friction that results from the usual technical back-and-forth of a vibrant open source project.

To get started with this process, visit the Sign the CLA page.
https://github.com/GNS3/cla/blob/master/sign-cla.md

For reference, the standard open source license under which GNS3 licenses its platform and software, including your contribution is the GNU GENERAL PUBLIC LICENSE version 3.

**Comment by noplay** [comment]
I will test it tomorrow but that sound good to me. Welcome aboard :)

**Comment by CapnCheapo** [comment]
Thanks, I'll clean up any remaining comments soon.  I have a request in for legal to look over the CLA to make sure there are no conflicts.

**Comment by noplay** [comment]
Legal fun :)

**Comment by noplay** [comment]
I pushed improvements here:
https://github.com/GNS3/gns3-gui/tree/CapnCheapo-remove_use_local

* Remove few non visible user_local 
* Rename General Settings to  Local settings in some screen to make them more clear

**Comment by noplay** [comment]
Merged thanks your first contribution is ready to ship!

**Comment by CapnCheapo** [comment]
Wow.  I didn't think that change would be that easy...

**Pull request closed without merging**